runelite-client/src/main/java/net/runelite/client/plugins/wintertodt/WintertodtConfig.java

```diff
@@ -128,4 +128,15 @@ default boolean notifyBrazierOut()
 	{
 		return true;
 	}
+
+	@ConfigItem(
+			position = 9,
+			keyName = "Overlay",
+			name = "Client Overlay Toggle",
+			description = "Toggle the overlay for this plugin"
```

Comment:
These lines are all indented one level too far.
```suggestion
		position = 9,
		keyName = "Overlay",
		name = "Client Overlay Toggle",
		description = "Toggle the overlay for this plugin"
```

Reply:
Also, the names given to the added config fields/method are not good. I would recommend naming the method and keyName `displayOverlay`, the name `Display Overlay`, and the description `Display an overlay which shows your activity status and number of points and kindling held in your inventory.`